src/clj/fluree/db/query/fql/parse.cljc

```diff
@@ -270,11 +270,15 @@
 
 (defn parse-where-clause
   [clause vars context]
-  (->> clause
-       util/sequential
-       (mapcat (fn [pattern]
-                 (parse-pattern pattern vars context)))
-       where/->where-clause))
+  ;; a single non-node where pattern is already sequential, so we need to specially handle it
+  (let [clause* (if (and (sequential? clause) (keyword? (first clause)))
+                  [clause]
+                  (util/sequential clause))]
+    (->> clause*
+         util/sequential
```

Comment:
This is no longer necessary since you've ensured that the clause is sequential above.

Reply:
I think you still need it if you've got a single node pattern, since that will fail the `sequential?` check above.

Reply:
Wait, nevermind.